skia-safe/examples/vulkan-window/renderer.rs

```diff
@@ -267,6 +280,7 @@ impl VulkanRenderer {
                 .swapchain
                 .recreate(SwapchainCreateInfo {
                     image_extent: window_size.into(),
+                    image_format: Format::B8G8R8A8_UNORM,
```

Comment:
This is redundant with the values pulled in on the next line via the existing swapchain's info, right?